ui/app/AppLayouts/Wallet/controls/SavedAddressesDelegate.qml

```diff
@@ -22,55 +22,64 @@ StatusListItem {
     property string name
     property string address
     property string ens
+    property string chainShortNames
     property bool favourite: false
-    property var saveAddress: function (name, address, favourite) {}
-    property var deleteSavedAddress: function (address) {}
+    property var saveAddress: function (name, address, favourite, chainShortNames, ens) {}
+    property var deleteSavedAddress: function (address, ens) {}
 
-    signal openSendModal()
+    signal openSendModal(string recipient)
 
-    implicitWidth: parent.width
+    implicitWidth: parent ? parent.width : 0
```

Comment:
@IvanBelyakoff this is just an assumption about parten, can we change this (remove this completely and set with from the place where it's used)?

Reply:
Just use
```qml
width: ListView.view.width
```
if it's indeed used inside a ListView